.eslintrc.yml

```diff
@@ -40,5 +40,18 @@ rules:
     - error
     - devDependencies: true
   padded-blocks: 0
-  key-spacing:
-    - 0
+  key-spacing: 0
+  space-before-function-paren:
+    - 2
+    - never
+  func-names:
+    - 2
+    - as-needed
+  react/prefer-stateless-function:
+    - 2
+    - ignorePureComponents: true
+  react/forbid-prop-types:
+    - 2
+    - forbid:
+      - any
+      - array
```

Comment:
By default it forbids `any`, `array`, and, `object` and the stub passes in an object. So I had to allow that one. I'm not sure how "bad" using proptype of `object` or `array` is. I can see not allowing `any` because we should be specific. Think we should allow `array` then?